servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -2349,6 +2391,54 @@ Policy updateTemplatePolicy(Policy policy, String domainName, String policyName,
         return templatePolicy;
     }
     
+    ServiceIdentity updateTemplateServiceIdentity(ServiceIdentity serviceIdentity, String domainName, String serviceIdentityName, List<TemplateParam> params) {
+        ServiceIdentity templateServcieIdentity = new ServiceIdentity().setName(ZMSUtils.serviceResourceName(domainName, serviceIdentityName));
```

Comment:
This is actually not correct. The name in the object must match to how we're returning json objects to clients - which is always in <domain>.<service> format.